index.html

```diff
@@ -531,10 +533,7 @@ <h3>The <dfn>constructor()</dfn> method</h3>
           Set |this|.{{PressureObserver/[[Callback]]}} to |callback:PressureUpdateCallback|.
         </li>
         <li>
-          If |options|["sampleRate"] is less than or equal to 0, throw a {{RangeError}}.
```

Comment:
0 is a valid value for unsigned long and unsigned long long. If is allowed here, it means users can make the "passes rate test" check always return true. It's not exactly wrong, so it's more like what you'd like to communicate here.

Reply:
It is a bit weird that it becomes a RangeError, so I more consider it the fastest frequency possible, which the implementation can limit

Reply:
Basically 0 means, give me events as soon/fast as possible